package/crossplane.yaml

```diff
@@ -1,4 +1,7 @@
 ---
+# TODO(negz): Update to v1 once we have a build of the crossplane
+# CLI with a fix for the below issue.
+# https://github.com/crossplane/crossplane/issues/5971
```

Comment:
I'd like to switch this function to v1 metadata so I can use it in the Crossplane E2E tests to test a fix for this issue. There's a chicken and egg situation though - I need to update `crossplane` (crank) before I can build the function.